### ISSUE TITLE
Survival world lags?

### ISSUE THREAD

**Issue by Creeperface01** [Nukkit/Nukkit#1510]
https://timings.aikar.co/?id=032085f3de5f4e67a685cda3f1ba3ba4

timing report from my server, idk if its a nukkit issue or plugin (MobPlugin)

**Comment by DaMatrix** [comment]
@Creeperface01 Are you using https://github.com/PocketDreams/SequinLand/pull/25 ? I had the same issue, and it was caused by that.

**Comment by Creeperface01** [comment]
no, i'll try it but can you tell me what part of nukkit's tick causes it?

**Comment by Creeperface01** [comment]
chunk loading and entity tick?

**Comment by Creeperface01** [comment]
i don't think 1 milisecond will solve this

**Comment by Creeperface01** [comment]
and ticking without sleep will cause permanent server 100% load won't it?

**Comment by Creeperface01** [comment]
https://timings.aikar.co/?id=90a5613f8aef4c96861abe9a00b383a1 without mobs

**Comment by Creeperface01** [comment]
and with 1 player

**Comment by DaMatrix** [comment]
Maybe data corruption? I can't think of any other reason unloading a chunk would take 500+ ms.

**Comment by Creeperface01** [comment]
its world generated in spigot, and it happens with any world i generate there

**Comment by Creeperface01** [comment]
i don't know what world should I use... nukkit world lags the client and PC world lags the server... 
one better than the other

**Comment by Creeperface01** [comment]
OK another report https://timings.aikar.co/?id=373f185c94b64840acea9f6e9b6c4e0f
world: https://www.dropbox.com/s/mknquyyatmqug0j/uhc-2.zip?dl=0
nukkit.yml: https://www.hastebin.com/uxucenuviz.bash

@Tee7even can you give me some advice?

**Comment by Creeperface01** [comment]
but it doesn't happen on all worlds, i thought it happens only on survival worlds but it happens even on my flat world

**Comment by Creeperface01** [comment]
@MagicDroidX @Snake1999 @Pub4Game @Tee7even can you say something to this?

**Comment by Creeperface01** [comment]
ok I'll try to add better timings to this part of nukkit tick and tell you better info what causes it

**Comment by sero583** [comment]
@DaMatrix Just by the way if it would occur on https://github.com/PocketDreams/SequinLand/pull/25, then it wouldn't be a Nukkit issue.

**Comment by Creeperface01** [comment]
can you send Existing link? :D

**Comment by sero583** [comment]
Ugh, I just copied and pasted what @DaMatrix wrote.
